### PR TITLE
Tycho 4.0.6

### PR DESCRIPTION
**Pull request by LorenzoBettini**
See https://github.com/eclipse/xtext/pull/2955

### CODE REVIEW

**Comment by LorenzoBettini** [comment]
@cdietrich we need Maven 3.9 for Tycho 4: what would you prefer? Adding the Maven Wrapper (and use that in the GitHub Actions) or use the action to install a specific version of Maven?

**Comment by LorenzoBettini** [comment]
Oh, we already use that action for some jobs; I'll update the failing job accordingly

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -116,8 +114,6 @@ jobs:
         with:
```
[the remark this answers is not in the document]
Comment (LorenzoBettini) [reply .github/workflows/build.yml]: Right! Done.

**Review by cdietrich — APPROVED** [review]

**Comment by LorenzoBettini** [comment]
@cdietrich I'll also remove your old branch

**Comment by LorenzoBettini** [comment]
sorry, wrong PR :)

**Pull request merged**